src/packet/signature/types.rs

```diff
@@ -713,7 +749,7 @@ impl SubpacketType {
             SubpacketType::SignatureTarget => 31,
             SubpacketType::EmbeddedSignature => 32,
             SubpacketType::IssuerFingerprint => 33,
-            SubpacketType::PreferredAead => 34,
+            SubpacketType::PreferredAead => 39,
```

Comment:
did this change? what about `34`?

Reply:
In RFC 9580, subpacket type 34 is "Reserved": https://www.rfc-editor.org/rfc/rfc9580.html#name-signature-subpacket-specifi

This change of ID is part of the IETF process' shift from v5 to v6, and vacating numbers that were previously used in 4880bis.